test/websockex_test.exs

```diff
@@ -1542,7 +1519,7 @@ defmodule WebSockexTest do
 
       new_server_pid = WebSockex.TestServer.receive_socket_pid()
       send(new_server_pid, :connection_continue)
-      ^new_server_pid = WebSockex.TestServer.receive_socket_pid()
+      new_server_pid = WebSockex.TestServer.receive_socket_pid()
```

Comment:
Why did we remove the pinned variable here?

Reply:
https://ninenines.eu/docs/en/cowboy/2.5/guide/ws_handlers/
_Post-upgrade initialization_ section is saying: 

> Cowboy has separate processes for handling the connection and requests. Because Websocket takes over the connection, the Websocket protocol handling occurs in a different process than the request handling.